src/components/PetCardComponent.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="pet-selection">
+    <div class="pet-card" @click="selectPet">
+      <img :src="pet.image" :alt="pet.name">
+      <div class="pet-card-name">
+        <p>{{ pet.name }}</p>
+        <img src="../assets/icon_audio.svg" alt="Audio">
+      </div>
+    </div>
+  </div>
+</template>
+
+
+<script setup lang="ts">
+import { defineProps, defineEmits } from 'vue';
+
+const props = defineProps<{
+  pet: { name: string; image: string };
```

Comment:
Export the Pet interface from petsConfig.ts and use it here instead of recreating the type.
{ `pet: Pet` }